python/tvm/relay/backend/contrib/tidl.py

```diff
@@ -1000,15 +980,22 @@ def tidl_import_dense(self, this_node):
         import_lib_dense(num_innodes, num_outnodes, ctypes.c_void_p(weights_array.ctypes.data))
         return True
 
-    def tidl_import_mul(self, this_node):
+    def tidl_import_max_mean(self, node):
+        r""" Import 'max' or 'mean' operator to TIDL pooling layer
+        """
 
-        mul_params = MulParams()
-        scale = this_node.args[0].data.asnumpy()
-        mul_params.scale = np.amax(scale)
-        import_lib_mul = self.import_lib.tidlImportMul
-        import_lib_mul.argtypes = (ctypes.POINTER(MulParams), ctypes.c_void_p)
-        import_lib_mul.restype = None
-        import_lib_mul(mul_params, ctypes.POINTER(ctypes.c_int)())
+        pooling_params = PoolingParams()
+        pooling_params.kernel_h = pooling_params.kernel_w = 0
+        pooling_params.pad_h = pooling_params.pad_w = 0
+        pooling_params.stride_h = pooling_params.stride_w = 1

```

Comment:
Does kernel width and height of 0 mean something special to the TIDL relay importer?

To directly map max/mean to pool, we would need to do the following (assuming NHWC):
1. If axis == [1]: kernel_h = args[0].checked_type.shape[1], kernel_w = 1
2. If axis == [2]: kernel_h = 1, kernel_w = args[0].checked_type.shape[2]
3. If axis == [1, 2]: kernel_h = args[0].checked_type.shape[1], args[0].checked_type.shape[2]

Then if `keepdims` is false, we need to add a squeeze/reshape.

https://tvm.apache.org/docs/api/python/relay/index.html#tvm.relay.max

Reply:
Yes, setting kernel width and height to 0 means this is a global average/max pool for TIDL. 
TIDL doesn't support keepdims being false. I will correct the annotation part.